internal/backup/volumesnapshotbackup_action.go

```diff
@@ -58,5 +65,39 @@ func (p *VolumeSnapshotBackupBackupItemAction) Execute(item runtime.Unstructured
 		return nil, nil, errors.WithStack(err)
 	}
 
-	return &unstructured.Unstructured{Object: vsbMap}, nil, nil
+	// Add temp VSClass as additional item
+	tempVSC := snapshotv1api.VolumeSnapshotClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: fmt.Sprintf("%s-snapclass", backup.Name),
+			Labels: map[string]string{
+				util.WaitVolumeSnapshotBackup: "true",
+			},
+		},
+		Driver:         "ebs.csi.aws.com",
```

Comment:
Should this be hardcoded to ebs? What about other providers?

Reply:
Good catch. I quickly did this for testing purposes and forgot to update it.

Reply:
A safer one IMO would be to copy an existing voluemsnapshotclass or get driver name from existing storageclass, change the name, and proceed. This would be a more generic way to achieve what you want across providers.